src/Files.App/ViewModels/Properties/SecurityViewModel.cs

```diff
@@ -139,18 +119,43 @@ public string DisableInheritanceOption
 			}
 		}
 
-		private async void DisableInheritance()
+		public RelayCommand EditOwnerCommand { get; set; }
+		public RelayCommand AddRulesForUserCommand { get; set; }
+		public RelayCommand RemoveRulesForUserCommand { get; set; }
+		public RelayCommand AddAccessRuleCommand { get; set; }
+		public RelayCommand RemoveAccessRuleCommand { get; set; }
+		public RelayCommand DisableInheritanceCommand { get; set; }
+		public RelayCommand<string> SetDisableInheritanceOptionCommand { get; set; }
+		public RelayCommand ReplaceChildPermissionsCommand { get; set; }
+		#endregion
+
+		#region Methods
+		private void InitializeCommands()
 		{
-			if (SetAccessRuleProtection(isProtected, preserveInheritance))
+			EditOwnerCommand = new RelayCommand(ChangeOwner, () => AccessControlListController is not null);
+			AddRulesForUserCommand = new RelayCommand(AddACE, () => AccessControlListController is not null && AccessControlListController.CanReadAccessControl);
+			RemoveRulesForUserCommand = new RelayCommand(RemoveACE, () => AccessControlListController is not null && AccessControlListController.CanReadAccessControl && SelectedAccessControlEntry is not null);
+			AddAccessRuleCommand = new RelayCommand(AddAdvancedACE, () => AccessControlListController is not null && AccessControlListController.CanReadAccessControl);
+			RemoveAccessRuleCommand = new RelayCommand(RemoveAdvancedACE, () => AccessControlListController is not null && AccessControlListController.CanReadAccessControl && SelectedAdvancedAccessControlEntries is not null);
+			DisableInheritanceCommand = new RelayCommand(DisableInheritance, () => AccessControlListController is not null && AccessControlListController.CanReadAccessControl && (AccessControlListController.IsAccessControlListProtected != _isProtected));
+			SetDisableInheritanceOptionCommand = new RelayCommand<string>(SetDisableInheritanceOption);
+			ReplaceChildPermissionsCommand = new RelayCommand(ReplaceChildPermissions, () => AccessControlListController is not null && AccessControlListController.CanReadAccessControl);
+		}
+
+		private void DisableInheritance()
+		{
+			if (SetAccessControlProtection(_isProtected, _preserveInheritance))
 			{
-				GetFilePermissions(); // Refresh file permissions
+				// Refresh file permissions
+				GetFilePermissions();
 			}
```

Comment:
I think comment should be placed outside of the `if`. You can also remove brackets